bundle-workflow/src/test_workflow/test_args.py

```diff
@@ -12,16 +12,35 @@
 
 
 class TestArgs:
-    manifest: str
+    s3_bucket: str
+    opensearch_version: str
+    build_id: int
+    architecture: str
+    test_run_id: int
     component: str
     keep: bool
+    logging_level: int
```

Comment:
How is logging_level `int`?

Reply:
Logging levels are mapped to integers internally by python.
Take a look at: https://docs.python.org/3/library/logging.html#logging.Logger.getEffectiveLevel